code/datums/components/fantasy/suffixes.dm

```diff
@@ -78,14 +77,76 @@
 	comp.appliedComponents += master.AddComponent(/datum/component/bane, picked_mobtype)
 	return "[newName] of [initial(picked_mobtype.name)] slaying"
 
+/datum/fantasy_affix/summoning
+	placement = AFFIX_SUFFIX
+	alignment = AFFIX_GOOD
+	weight = 5
+
+/datum/fantasy_affix/summoning/apply(datum/component/fantasy/comp, newName)
+	. = ..()
+	// This is set up to be easy to add to these lists as I expect it will need modifications
+	var/static/list/possible_mobtypes
+	if(!possible_mobtypes)
+		// The base list of allowed mob/species types
+		possible_mobtypes = typecacheof(list(
+			/mob/living/simple_animal,
+			/mob/living/carbon,
+			/datum/species,
+			))
+		// Some particular types to disallow if they're too broad/abstract
+		possible_mobtypes -= list(
+			/mob/living/simple_animal/hostile,
+			)
+		// Some types to remove them and their subtypes
+		possible_mobtypes -= typecacheof(list(
+			/mob/living/carbon/human/species,
+			/mob/living/simple_animal/hostile/megafauna,
+			))
+
+	var/mob/picked_mobtype = pick(possible_mobtypes)
+	// This works even with the species picks since we're only accessing the name
+
+	var/obj/item/master = comp.parent
+	var/max_mobs = CEILING(comp.quality/2, 1)
+	var/spawn_delay = 300 - 30 * comp.quality
+	comp.appliedComponents += master.AddComponent(/datum/component/summoning, list(picked_mobtype), 100, max_mobs, spawn_delay)
+	return "[newName] of [initial(picked_mobtype.name)] summoning"
+
+/datum/fantasy_affix/shrapnel
+	placement = AFFIX_SUFFIX
+	alignment = AFFIX_GOOD
+
+/datum/fantasy_affix/shrapnel/apply(datum/component/fantasy/comp, newName)
+	. = ..()
+	var/static/list/possible_projectiletypes
+	if(!possible_projectiletypes)
+		// The base list of allowed types
+		possible_projectiletypes = typecacheof(list(
```

Comment:
This should probably be hand picked and not just every kind, they have a *lot* of variety and many will break things.

This also lets you weight them if you want